man/dnstable_query.3.txt

```diff
@@ -110,6 +110,9 @@ from the results.  The default is 0, which means to not offset any rows.
 ^dnstable_query_set_aggregated^() sets if the results from queries should be
 aggregated. The default is _true_ for aggregated results.
 
+^dnstable_query_set_case_sensitive() specifies that query parameters should be
+treated as case-sensitive.
+
```

Comment:
This should note that the case is preserved for hostnames added _after_ the call to `dnstable_query_set_case_sensitive()` and will have no effect if called after `dnstable_query_set_data()`